template/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "your-app-name",
```

Comment:
Name MUST be lowercase version of project name.

Reply:
Added package-json-handler which will rename in package.json file `name` and `version` if is provided in a command line.